src/Model.hs

```diff
@@ -34,7 +34,7 @@ User json
   Id Int64
   name Text
   passwordHash BCrypt
-  apiToken Text Maybe
+  apiToken HashedApiKey Maybe
```

Comment:
It would probably be better to have a separate `api_keys` table that corresponds keys with users, so that a user can have multiple keys created for possibly different purposes. Though this field was already present (albeit unused), so for the initial implementation I think this is fine.